package.json

```diff
@@ -34,12 +34,16 @@
 	"scripts": {
 		"build": "yarn clean && lerna run build --stream",
 		"clean": "lerna run clean",
-		"ganache-cli": "npx ganache-cli -m 'smart guide what forget tired jungle always expire rescue boring glue champion'",
+		"ganache-cli": "npx ganache-cli -m 'smart guide what forget tired jungle always expire rescue boring glue champion' --account=\"0x4c3758228f536f7a210f8936182fb5b728046970b8e3215d0b5cb4c4faae8a4e,100000000000000000000\" --account=\"0x34aeb1f338c17e6b440c189655c89fcef148893a24a7f15c0cb666d9cf5eacb3,100000000000000000000\"",
+		"ganache-cli-background": "yarn ganache-cli > /dev/null & client=$! npx wait-port 8545",
+		"geth:build": "docker build ./.github -t gethnode -f .github/Dockerfile",
+		"geth": "docker run -d -p 8545:8545 gethnode --datadir data --networkid 15 --http.addr 0.0.0.0 --http.port 8545 --http",
```

Comment:
From [1.x](https://github.com/ChainSafe/web3.js/blob/d46d922e2d9714b6e8bd72db665f708f12e222ea/package.json#L30):

```javascript
"geth": "geth-dev-assistant --accounts 5 --tag stable --gasLimit 7000000",
```

Reply:
we will add geth-dev-assistant as a third option of Eth Node later

Reply:
`geth-dev-assistant` is just a tool that makes initializing a network easier, there's no reason to offer this as a separate option

Reply:
I've built our geth docker with our own genesis.json file which has accounts with a balance. We will use them to test all things.